process/transaction/metaProcess.go

```diff
@@ -145,6 +145,8 @@ func (txProc *metaTxProcessor) ProcessTransaction(tx *transaction.Transaction) (
 		if txProc.enableEpochsHandler.IsFlagEnabled(common.ESDTFlag) {
 			return txProc.processSCInvoking(tx, tx.SndAddr, tx.RcvAddr)
 		}
+	case process.RelayedTxV3:
```

Comment:
can you confirm that the txType detected here is one of the above : SCDeployment, SCInvoking, BuiltInFunctionCall.

I don't think we need a process.RelayedTxV3 here.